src/main/java/net/sf/jabref/gui/search/SearchResultsDialog.java

```diff
@@ -480,7 +481,7 @@ public Object getColumnValue(BibEntry entry, int column) {
             }
             else {
                 String field = FIELDS[column - PAD];
-                if ("author".equals(field) || "editor".equals(field)) {
+                if (InternalBibtexFields.getFieldExtras(field).contains(BibtexSingleFieldProperties.PERSON_NAMES)) {
```

Comment:
I think it would make sense to add a method `isPersonName(field)` to InternalBibtexFields since this is check occurs quite often.